Cargo.toml

```diff
@@ -340,7 +340,7 @@ tokio-postgres = { version = "0.7.10", default-features = false, features = ["ru
 tokio-tungstenite = {version = "0.20.1", default-features = false, features = ["connect"], optional = true}
 toml.workspace = true
 tonic = { version = "0.10", optional = true, default-features = false, features = ["transport", "codegen", "prost", "tls", "tls-roots", "gzip"] }
-hickory-proto = { version = "0.24.1", default-features = false, features = ["dnssec"], optional = true }
+hickory-proto = { git = "https://github.com/hickory-dns/hickory-dns.git", rev = "107635c6c5934524894736f1b141198d0fa62fec", default-features = false, features = ["dnssec"], optional = true }
```

Comment:
Could we add a comment similar to the one you have in the PR description about why we are pinning to a rev?

Reply:
Added, but I'm not sure if we should wait until 0.25 is released.

Reply:
> Added, but I'm not sure if we should wait until 0.25 is released.

Thanks! I tend to agree that maybe we should wait for the next release though since it seems to be in-progress and this is a feature addition rather than fixing a bug. Would that be alright with you?

Reply:
@jszwedko upgraded to `0.25.0-alpha.1` and it should be working now.

Reply:
@jszwedko upgraded to 0.25.0-alpha.1 and it should be working now.